README.md

```diff
@@ -0,0 +1,157 @@
+# 📚Book Shop📚
+Welcome to the Online Bookstore API! This API provides essential
+functionalities for managing books, user authentication,
+and order processing, forming the backbone of a scalable and
+efficient online bookstore. Inspired by the need for a
+robust solution
+in the digital book retail space, this project showcases
+a fully functional online
+bookstore built with Spring Boot. It includes features
+like user registration, book browsing, shopping cart
+management, and order processing, offering a solid foundation
+for real-world applications.
+
+## 🛠️Technologies and Tools🛠️
+-**Spring Boot**: Core framework for building application.
+
+-**Spring Security**: Ensures robust authentication and authorization mechanisms for the application.
+
+-**Spring Data JPA**: Simplifies database interactions through JPA.
+
+-**Swagger**: Offers interactive API documentation and testing capabilities.
+
+-**JUnit**: Enables efficient unit testing.
+
+-**Mockito**: Aids in isolating code under test by mocking its dependencies, facilitating effective unit testing.
+
+-**MYSQL Database**: For the relational database.
+
+-**Liquibase**: Manages database migrations.
+
+## Features and Functionality
+### Authentication
+- **Register user**: `POST /api/auth/register`
+- **Login user**: `POST /api/auth/login`
```

Comment:
add more details to this, how does auth work in your app, give requests examples on how to register/login/use the provided token, tell about the 2 roles you have in your app, add credentials for user and admin role users so that the visitor of your project could test admin-only endpoints as well, because it is a demonstration project after all :)

Maybe even add a separate topic for all of the things that I mentioned here